tree.scss

```diff
@@ -179,8 +162,17 @@ $timing-fn: ease-out;
     &.transitions {
       ul {
         li.node {
+          @include transition-property(transform, opacity, height);
           &.transitioning-node {
+            // Initially set the opacity to 0 for incoming nodes so they can transition to 1. 
+            // Set to 0 for outgoing nodes
             opacity: 0;
+            // Don't apply transition on opacity here. Otherwise, IE11 has a bug that will break the entire transition
+            // Instead, apply the transition to the incoming/outgoing classes. 
+            @include transition-property(height, transform); 
```

Comment:
@nathanbowser This line is pretty much the only actual thing that changed from how it originally worked. And that is only for a split second since its applied and instantly removed.